packages/edit-post/src/components/sidebar/post-status/index.js

```diff
@@ -64,12 +64,12 @@ export default function PostStatus() {
 						<PostSchedulePanel />
 						<PostTemplate />
 						<PostURLPanel />
+						<PostSyncStatus />
 						<PostSticky />
 						<PostPendingStatus />
 						<PostFormat />
 						<PostSlug />
 						<PostAuthor />
-						<PostSyncStatus />
```

Comment:
I moved this because it actually looks way better before the "pending status". Check by opening a reusable block in the post editor.

Reply:
I'm not sure how to view this component 😅

Reply:
You click on the "manage all patterns" link in the more menu of the post editor and then create a pattern/reusable block from there.